vertical-pod-autoscaler/pkg/admission-controller/resource/pod/patch/resource_updates_test.go

```diff
@@ -164,11 +164,32 @@ func TestClalculatePatches_ResourceUpdates(t *testing.T) {
 				addAnnotationRequest([][]string{{cpu}}, request),
 			},
 		},
+		{
+			name: "replacement cpu request recommendation from container status",
+			pod: test.Pod().
+				AddContainer(core.Container{}).
```

Comment:
To reflect a more real scenario, perhaps we should set a value here for container requests?

Then we can show that it is make changes based on the container status value.

Reply:
Here the value of the resources doesn't matter, but the presence/absence.  

For this specific test case, if the pod spec has empty resource requests, we should also expect an add resource/request patch. So, without resources in containerStatus, we should've expected 2 more patches (like the "new cpu recommendation" test case).